users/userList.js

```diff
@@ -1,23 +1,41 @@
+const email = require("../helpers/email");
+
 module.exports = function makeUserList(db_model) {
+    return Object.freeze({
+        add,
+        remove,
+        findOneByEmail,
+        updateEmail,
+        updateFirstName,
+        updateLastName,
+    });
+
+    async function add(user) {
+        return await db_model.create(user);
+    }
 
-	return Object.freeze({
-		add,
-		remove,
-		findOneByEmail,
-	});
+    async function remove(user = {}) {
```

Comment:
This standard seems a bit dangerous, defaulting to remove all users if no attribute is received.